presto-orc/src/main/java/com/facebook/presto/orc/metadata/OrcFileTail.java

```diff
@@ -80,4 +85,23 @@ public int getMetadataSize()
     {
         return metadataSize;
     }
+
+    public Optional<DwrfStripeCacheData> getDwrfStripeCacheData()
+    {
+        return dwrfStripeCacheData;
+    }
+
+    public int getDwrfStripeCacheSize()
+    {
+        int dwrfStripeCacheSize = 0;
+        if (dwrfStripeCacheData.isPresent()) {
+            dwrfStripeCacheSize = dwrfStripeCacheData.get().getDwrfStripeCacheSize();
+        }
+        return dwrfStripeCacheSize;
+    }
+
+    public int getTotalSlicesSize()
```

Comment:
should we just call this getTotalSlice.

Reply:
I'n not following. Did you meant getTotalSize?

Reply:
Sorry, my bad, I mean getTotalSize.